src/Exporter.php

```diff
@@ -69,7 +74,16 @@ public function shortenedRecursiveExport(array &$data, ?RecursionContext $proces
         /* @noinspection UnusedFunctionResultInspection */
         $processed->add($data);
 
+        $i     = 0;
+        $count = count($data, COUNT_RECURSIVE);
```

Comment:
```suggestion
        $count = count($array);
```

I think the (supressed) elements in the shortened array should be reported instead.

Reply:
added a test

Reply:
I personally would limit it by like 10 top elements and max. 10 sub elements. Otherwise the usability value of such shortened output is very low, imagine first element with large subarray.

Reply:
Please suggest a test case which is nit covered atm